src/JWT/JWT.csproj

```diff
@@ -43,5 +43,8 @@
 
   <ItemGroup Condition="'$(TargetFramework)' == 'netstandard1.3'">
     <PackageReference Include="System.Security.Cryptography.Csp" Version="4.3.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.5.0-preview-20170810-02" />

```

Comment:
Do you know why both MsTest and xUnit are here?

Reply:
oh sorry. that was a test, to get the test runing. I remove this.

Reply:
done